### PR TITLE
[Feature Request] New CBR policy sdk supported

### PR DESCRIPTION
**Pull request by Lance52259**
**What this PR does / why we need it**:
According to existing demands, it is now necessary to support CBR policy related interface functions.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: 
```
NONE
```

**Special notes for your reviewer**:

**Release note**:

```release-note
1. new CBR policy sdk supported
2. new sdk test supported
```

## Acceptance Steps Performed
```
go test -v -run Test=== RUN   TestCreateV3PolicyMarshall
--- PASS: TestCreateV3PolicyMarshall (0.00s)
=== RUN   TestCreateV3Policy
--- PASS: TestCreateV3Policy (0.00s)
=== RUN   TestDeleteV3Policy
--- PASS: TestDeleteV3Policy (0.00s)
=== RUN   TestUpdateV3Policy
--- PASS: TestUpdateV3Policy (0.00s)
PASS
ok      github.com/huaweicloud/golangsdk/openstack/cbr/v3/policies/testing      0.014s
```

### CODE REVIEW

File: openstack/cbr/v3/policies/urls.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package policies
+
+import "github.com/huaweicloud/golangsdk"
+
+func listURL(client *golangsdk.ServiceClient) string {
```
Comment (ShiChangkuo) [inline openstack/cbr/v3/policies/urls.go:5]: s/`listURL`/`rootURL`/

File: openstack/cbr/v3/policies/urls.go [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+	return client.ServiceURL("policies")
+}
+
+func singleURL(client *golangsdk.ServiceClient, id string) string {
```
Comment (ShiChangkuo) [inline openstack/cbr/v3/policies/urls.go:9]: s/`singleURL`/`resourceURL`/

File: openstack/cbr/v3/policies/urls.go (hunk 1 above)
Comment (Lance52259) [reply openstack/cbr/v3/policies/urls.go]: updated

File: openstack/cbr/v3/policies/urls.go (hunk 2 above)
Comment (Lance52259) [reply openstack/cbr/v3/policies/urls.go]: updated

**Pull request merged**